plugins.md

```diff
@@ -11,10 +11,10 @@ Plugins
 
 <em class="t-gray">Extensions that use the RubyGems plugin API.</em>
 
-As of RubyGems 1.3.2, RubyGems will load plugins installed in gems or
-`$LOAD_PATH`.  Plugins must be named 'rubygems\_plugin' (.rb, .so, etc)
-and placed at the root of your gem's #require\_path.  Plugins are
-discovered via `Gem::find_files` then loaded.
+RubyGems will load plugins in the latest version of each installed gem or
+`$LOAD_PATH`.  Plugins must be named 'rubygems\_plugin' (.rb, .so, etc) and
+placed at the root of your gem's #require\_path.  Plugins are installed at a
+special location and loaded on boot.
```

Comment:
For comparison, here is the corresponding text from the change in the plugin's PR:

```
# RubyGems will load plugins in the latest version of each installed gem or
# $LOAD_PATH.  Plugins must be named 'rubygems_plugin' (.rb, .so, etc) and
# placed at the root of your gem's #require_path.  Plugins are installed at a
# special location and loaded on boot.
```

https://github.com/rubygems/rubygems/blob/a28724fdd9262d716e6facb1950387a0a6ac4c73/lib/rubygems.rb#L45-L48